teamB/simchaewon/models/likeDao.js

```diff
@@ -0,0 +1,36 @@
+const { DataSource } = require("typeorm");
+
+const westa_DB = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  password: process.env.TYPEORM_PASSWORD,
+  port: process.env.TYPEORM_PORT,
+  database: process.env.TYPEORM_DATABASE,
+  username: process.env.TYPEORM_USERNAME,
+});
+
+westa_DB.initialize()
+
+  .then(() => {
+    console.log("DATA SOURCE HAS BEEN INITIALIZED!");
+  })
+  .catch((err) => {
+    console.error("Error ocurred during Data Source initialization", err);
+    westa_DB.destroy();
+  });
+
+  const clickLike=async(userId, postId)=>{
+    try{
+        return await westa_DB.query(`INSERT INTO likes (user_id, post_id) VALUES (?, ?);`,
+        [userId, postId]
+        );
```

Comment:
- SQL 쿼리문 포맷팅을 맞춰서 수정해주세요 !
```suggestion
        return await westa_DB.query(`
        INSERT INTO likes (
            user_id, 
            post_id
        ) VALUES (?, ?);`,
        [userId, postId]
        );
```